odoo/addons/base/views/res_partner_views.xml

```diff
@@ -155,7 +155,7 @@
             <field name="arch" type="xml">
                 <form string="Partners">
                 <div class="alert alert-warning oe_edit_only" role="alert" attrs="{'invisible': [('same_vat_partner_id', '=', False)]}">
-                  A partner with the same <span><span class="o_vat_label">Tax ID</span></span> already exists (<field name="same_vat_partner_id"/>), are you sure to create a new one?
+                  A partner with the same <span><span class="o_vat_label"><field name="vat_label"/></span></span> already exists (<field name="same_vat_partner_id"/>), are you sure to create a new one?
```

Comment:
@fpodoo to change this Tax ID to the related vat label then we need compute field. so we improve it in V16 or not?